schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -991,7 +1200,6 @@
           "description": "The administrator login password (required for server creation)."
         }
       },
-      "required": [ "administratorLogin", "administratorLoginPassword" ],
```

Comment:
These are required for creating a server, but not for updating. I made this required earlier so that it's clear they are required for create, but now I understand that update must also be supported so they should not be required.